src/app/services/navbar.service.ts

```diff
@@ -47,11 +48,18 @@ export class NavbarService {
       this.activeItem = this._items[0];
     } else if (this.isAnalysis) {
       this.activeItem = this._items[1];
+      this.configFractalis();
     } else if (this.isExport) {
       this.activeItem = this._items[2];
     }
   }
 
+  private configFractalis() {
+    if (!this.fractalisService.F) {
```

Comment:
Seems better not to expose the `F` object, but to encapsulate it in the fractalis service. Having a public property `isInitialised` or something is better and more to the point.